composer.json

```diff
@@ -19,8 +19,8 @@
         "php": ">=5.6.0",
         "behat/behat": "~3.0",
         "atoum/atoum": "~1.0|~2.0|~3.0|3.0.x-dev",
-        "symfony/property-access": "~2.4|~3.0",
-        "justinrainbow/json-schema": ">=3.0 <=6.0",
+        "symfony/property-access": "~2.5|~3.0",
+        "justinrainbow/json-schema": ">=3.0 <6.0",
```

Comment:
Oops, in the previous PR I did `<=6.0.0` this was a big mistake 👼

Reply:
:scream: